deployment/amplify/staging/amplify/.config/project-config.json

```diff
@@ -1,14 +1,14 @@
 {
```

Comment:
As you can see from the filename change, `taui/src/amplify` got moved to `deployment/amplify/staging`, while `deployment/amplify/production` stores the production deployment.